test/conformance/conformance_task_group.cpp

```diff
@@ -129,7 +131,12 @@ TEST_CASE("Task handle being non copyable"){
 //! Test that task_handle prolongs task_group::wait
 //! \brief \ref requirement
 TEST_CASE("Task handle blocks wait"){
+    oneapi::tbb::global_control s(oneapi::tbb::global_control::max_allowed_parallelism, 1);
```

Comment:
Is it debug stuff or would you like to address TODO about adding more accurate test below?

Reply:
TODO now is addressed, some comments added